WikiLookup.json

```diff
@@ -575,5 +575,46 @@
 			"Star Citizen",
 			"Squadron 42"
 		]
+	},
+		{
+		"name": "Unofficial Elder Scrolls Pages",
+		"homepage": "https://en.uesp.net	",
+		"lang": "en",
+		"api": "https://en.uesp.net/w/api.php",
+		"companies": [
+		    "Bethesda Softwarks",
+			"Zenimax Online Studios"
+		],
+		"games": [
+		    "Arena",
```

Comment:
Please use consistent spacing throughout. Lines should only use tabs.

Reply:
Also, please use the complete names for the games, such as `The Elder Scrolls: Arena`.